shell/common/node_bindings.cc

```diff
@@ -169,6 +170,30 @@ void V8FatalErrorCallback(const char* location, const char* message) {
   *zero = 0;
 }
 
+void V8OOMErrorCallback(const char* location, const v8::OOMDetails& details) {
+  LOG(ERROR) << (details.is_heap_oom ? "javascript" : "process")
```

Comment:
For parity let's use something closer to what Node.js uses:

https://github.com/codebytere/node/blob/52ac44888df66df91df921e307b309eceb360f4c/src/node_errors.cc#L600-L602
```
  const char* message =
      details.is_heap_oom ? "Allocation failed - JavaScript heap out of memory"
                          : "Allocation failed - process out of memory";
```